recipes/mapserver/meta.yaml

```diff
@@ -0,0 +1,77 @@
+# NB: Lowercase!
+{% set name = "mapserver" %}
+{% set version = "7.6.4" %}
+
+package:
+  name: {{ name }}
+  version: {{ version }}
+
+source:
+  url: https://download.osgeo.org/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: b46c884bc42bd49873806a05325872e4418fc34e97824d4e13d398e86ea474ac
+
+build:
+  number: 0
+  # Any volunteers? :)
```

Comment:
:rofl:

Reply:
I was "volunteered" 😄

Reply:
@hobu You volunteered to setup the windows build?  Oh, that is kind of you :)